ci/infra/testrunner/tests/test_deployments.py

```diff
@@ -0,0 +1,33 @@
+from platforms import Platform
+from skuba import Skuba
+from kubectl import Kubectl
+import pytest
+import time
+from timeout_decorator import timeout
+
+@pytest.fixture
```

Comment:
This fixture should be moved to the test library `tests/conftest.py`and re-used in both the deployment and worker tests. Can you add a commit to do so?